cli/flox-watchdog/src/logger.rs

```diff
@@ -40,3 +48,245 @@ pub(crate) fn init_logger(file_path: &Option<PathBuf>) -> Result<(), anyhow::Err
         .init();
     Ok(())
 }
+
+/// Starts a background thread which emits a log entry at an interval. This is
+/// used as an indication of whether a watchdog's log file can be garbage
+/// collected. The thread will run until the watchdog exits.
+pub(crate) fn spawn_heartbeat_log() {
+    spawn(move || loop {
```

Comment:
suggestion: remove `move` since nothing is moved into the thread anymore
```suggestion
    spawn(|| loop {
```

Reply:
Done